server/test/unit/server/webservices/test_application.py

```diff
@@ -0,0 +1,20 @@
+import unittest
+
+from mock import patch, Mock
+
+from pulp.server.webservices.application import wsgi_application
+from pulp.server.initialization import InitializationException
+
+class TestApplication(unittest.TestCase):
+
+    @patch('pulp.server.webservices.application._initialize_pulp')
+    def test_wsgi_application_exception(self, mock_initialize_pulp):
+        mock_initialize_pulp.side_effect = Exception
+        with self.assertRaises(Exception):
```

Comment:
I suggest using `self.assertRaises(Exception, wsgi_application)` instead of the 'with' operator.